vowpalwabbit/test_common/include/vw/test_common/test_common.h

```diff
@@ -6,14 +6,23 @@
 
 #include "vw/config/options.h"
 #include "vw/config/options_cli.h"
+#include "vw/core/vw.h"
+#include "vw/json_parser/parse_example_json.h"
 
 #include <vector>
 
 namespace vwtest
 {
+constexpr float FLOAT_TOL = 0.0001f;
```

Comment:
This shouldn't be needed in our test code anymore. Places which used to use it can become `EXPECT_FLOAT_EQ`